File: JavaScript/03-Array-Programs/reverse-array.js
function reverseArr(arr) {
  let reverse = [];

  // for(let i = arr.length-1; i>= 0; i--){
  //     reverse.push(arr[i])
  // }
  
  let j = 0;
  
  for (let i = arr.length - 1; i >= 0; i--) {
      reverse[j] = arr[i];
      j++;
    }
    console.log(reverse);
}
reverseArr([23, 12, 3, 45, 3]);

File: 02-Array-Programs/02-Array-Programs/ReverseArray.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _02_Array_Programs
{
    internal class ReverseArray
    {
        void reverse(int[] arr)
        {
            Array.Reverse(arr);
            foreach (int i in arr)
            {
                Console.Write(i+" ");
            }
        }

        public static void Main(string[] args)
        {
            ReverseArray r = new ReverseArray();
            r.reverse(new int[] { 1, 2, 3, 4, 5, 6 });
        }
    }
}
